source/utilities.cc

```diff
@@ -906,11 +913,14 @@ namespace aspect
       return return_value;
     }
 
-
-    std::pair<double,double> real_spherical_harmonic( const unsigned int l,
-                                                      const unsigned int m,
-                                                      const double theta,
-                                                      const double phi)
+    //Evaluate the cosine and sine terms of a real spherical harmonic.
+    //This is a fully normalized harmonic, that is to say, inner products
+    //of these functions should integrate to a kronecker delta over
```

Comment:
this change is unrelated to your PR (and we also only keep the comments in the header file). Can you undo the change, please?